<?xml version="1.0" encoding="UTF-8"?>
<launch>

<arg name="color_topic"     default="toColorTopic"/>
<arg name="depth_topic" 	default="toDepthTopic"/>
<arg name="ir_topic" 	    default="toIrTopic"/>

<node pkg="cam_rtsp_vik" type="cam_rtsp" name="cam_rtsp" output="screen"/>
    <param name="_color_topic" 		value="$(arg color_topic)"/>
    <param name="_depth_topic" 	    value="$(arg depth_topic)"/>
    <param name="_ir_topic" 		value="$(arg ir_topic)"/>
</launch>